rados/rados_read_op_assert_version.go

```diff
@@ -0,0 +1,22 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package rados
+
+// #cgo LDFLAGS: -lrados
+// #include <rados/librados.h>
+// #include <stdlib.h>
+//
+import "C"
+
+// Ensure that the object exists and that its internal version number is equal
+// to “ver” before reading. “ver” should be a version number previously
```

Comment:
By looking at the JSON version of the comment I see that these are not "ascii" double-quote characters. Do you mind please changing these to the more common double-quote characters (" aka \x22), please?